client/src/js/dd.js

```diff
@@ -1,3 +1,2 @@
-requirejs([ './common' ], function(common) {
-  requirejs([ '../js/dd_main' ]);
-});
+requirejs([ './common' ],
+          function(common) { requirejs([ '../src/js/dd_main' ]); });
```

Comment:
I'm pretty sure with our current server setup, since it's serving `src/` as ``localhost:8080/``, you want this to be ``../js/dd_main`` or just ``/js/dd_main``